doc/source/contributing/development_plan.rst

```diff
@@ -1,45 +1,51 @@
+This page provides a high-level overview of some focus areas of ongoing and for future icepyx development.
+This list is not exclusive or highly specific.
+Instead, specific development tasks and new functionality implementations are driven by individual developers/teams.
 
-Items with a smaller scope are tracked as issues on our GitHub `issue tracker <https://github.com/icesat2py/icepyx/issues>`_. We invite you to join the active discussions happening there.
+Our ongoing efforts are tracked as issues on our GitHub `issue tracker <https://github.com/icesat2py/icepyx/issues>`_.
+We invite you to join the active discussions happening there.
 
-Enhancing User Interactivity and Visualization
-----------------------------------------------
-The process of querying, obtaining, and working with ICESat-2's large datasets through a command line or similar interface poses challenges to many researchers who are uninterested in also becoming advanced software developers. However, downloading, storing, and backing up both raw and derived data are computationally and resource intensive, and frequent switching between tools for different steps in the research workflow are time intensive and difficult to reproduce. By simplifying the process of querying, subsetting, and visualizing data - both through relevant function methods and interactive tools like Jupyter widgets - icepyx aims to reduce or remove the need to download large, non-subsetted datasets, enable easy visualization throughout the data inquiry to analyzed data presentation steps, and provide a simple, community-based framework for reproducibility.
+Major Themes for Development
+----------------------------
 
-Improving Accessibility to Advanced Computing
----------------------------------------------
-Even as new resources and tools are developed to make computing easier, disciplinary researchers still face challenges finding time to maintain their computing environments while also meeting their research, teaching, and service objectives. Through integration of icepyx into the Pangeo ecosystem, only a handful of developers are able to manage the computational resources needed for working with ICESat-2 data, freeing researcher time and energy for exploring data. Further, the integration of Pangeo into existing advanced computing infrastructure (such as NASA's ADAPT) will enable researchers to take advantage of cloud computing without a significant need for re-tooling.
+Enhancing User Interactivity and Visualization
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+icepyx aims to continually reduce the need for researchers to rewrite "routine" code by enabling easy visualization
+throughout the data inquiry to analyzed data presentation process and provide a simple, community-based framework for reproducibility.
```

Comment:
Might need to wordsmith this a bit more.
```suggestion
in an end-to-end manner from the process of data inquiry to the presentation of the analyzed data,
and provide a simple, community-based framework for reproducibility.
```

Reply:
I like this change